_episodes/12-cmdline.md

```diff
@@ -118,6 +124,8 @@ Whenever Python runs a program,
 it takes all of the values given on the command line
 and puts them in the list `sys.argv`
 so that the program can determine what they were.
+Our script name is the first argument after `python` on the
+command line, therefore it is assigned to the variable argv[0].
```

Comment:
I am not sure if this adds to the `sys.argv[0]` explanation below. It may be confusing that here it is referring to the script name as an argument, but [`sys.argv`](https://docs.python.org/3/library/sys.html#sys.argv) is explained as the arguments passed to the script (implying that though the path or name is in `sys.argv[0]` it is not considered an argument in the context of arguments we are passing to the script we are calling). Does that make sense?

Reply:
You know, I don't really know, but thanks for addressing the comments!